json-rpc/src/tests/unit_tests.rs

```diff
@@ -264,10 +264,74 @@ proptest! {
         assert_eq!(data.get("version").unwrap().as_u64(), Some(actual_version));
         assert_eq!(data.get("timestamp").unwrap().as_u64(), Some(actual_timestamp));
     }
+
+    #[test]
+    fn test_get_transactions(blocks in arb_blocks_to_commit()) {
+        _test_validate_transactions(blocks);
+    }
+}
+
+fn _test_validate_transactions(blocks: Vec<(Vec<TransactionToCommit>, LedgerInfoWithSignatures)>) {
```

Comment:
why the _ before test? it is already explicitly private.

Reply:
fixed.